src/pages/Admin/Venue/Rooms/RoomsForm.tsx

```diff
@@ -320,29 +321,19 @@ const RoomInnerForm: React.FC<RoomInnerFormProps> = (props) => {
                   </div>
                   <div className="toggle-room">
                     <div className="input-title">Enabled ?</div>
-                    <label className="switch">
-                      <input
-                        disabled={disable}
-                        type="checkbox"
-                        id="isEnabled"
-                        name={"isEnabled"}
-                        ref={register}
-                      />
-                      <span className="slider round"></span>
-                    </label>
+                    <Toggler
+                      id="isEnabled"
+                      forwardedRef={register}
+                      disabled={disable}
+                    />
                   </div>
                   <div className="toggle-room">
                     <div className="input-title">Is label hidden?</div>
```

Comment:
Assuming we use the `label` prop on the `Toggler`, this will be redundant:
```suggestion
```

Reply:
Would prefer subsequent PR dealing with labels due to possible styling issues. More at https://github.com/sparkletown/sparkle/pull/1775/files#r667330479

Reply:
See my thoughts on that in https://github.com/sparkletown/sparkle/pull/1775/files#r667331265

**tl;dr** Let's see how hard it will be to fix here, and if we can do it all at once, then we should. As that is really part of converting them to use the `Toggler` component.

Reply:
Added `@debt` in 922a6c35760b6eda64d271b736bc43c732999db0 for future PR